packages/common/src/types/warehouse.ts

```diff
@@ -50,7 +50,7 @@ export interface WarehouseClient {
         streamCallback: (data: WarehouseResults) => void,
         options: {
             values?: any[];
-            tags?: Record<string, string>;
+            tags: Record<string, string>;
```

Comment:
the ticket also mentions `runQuery` ( down in line 66 ⬇️ ) to have `tags` required, could you make the change @Ankush263 ?

I'm closing the PR for now, but feel free to reopen/create another one with the 2 requirements on the main issue 👍

Reply:
ok, I have raised a PR #11673 to resolve it.